### DISCUSSION TITLE
Sign Out 후 다시 Sign In 시 로그인되지 않음

### DISCUSSION THREAD

**Discussion by IronKoh** [metamong-friends/metamong-friends.github.io#49] [제보하기]
메타몽 앱을 실행하여 로그인 후 스페이스 / 캐릭터 선택 창에서
Sign Out 버튼을 누르고, 다시 로그인을 시도하면
로그인이 되지 않습니다.
(아래 화면에서 멈추고 로그인이 안됩니다.)

앱을 종료 후 다시 실행하여 로그인하면 잘 됩니다.

![image](https://github.com/metamong-friends/metamong-friends.github.io/assets/169414893/8767cfd0-ff21-4a54-a041-2d254214f0d9)

**Comment by ACT-TJ** [comment]
안녕하세요!

오늘 접속이 안되어 많이 힘드셨던 점 죄송한 말씀드립니다.
화면공유로 직접 같이 확인하였고, 빠르게 조치할 수 있도록 
담당자 확인하여 처리 진행하겠습니다!

**Comment by shlee303** [comment]
@IronKoh 해당 버그 조치 완료 하였습니다! 좋은 제보 감사합니다!
